pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiCompositeQuestion.java

```diff
@@ -64,6 +64,14 @@ void insertChild(@Bind("parentQuestionId") long parentQuestionId, @Bind("childQu
     @UseRowReducer(RowReducer.class)
     Optional<CompositeQuestionDto> findParentDtoByChildQuestionId(@Bind("childQuestionId") long childQuestionId);
 
+    @UseStringTemplateSqlLocator
+    @SqlQuery("queryCompositeQuestionIdByChildQuestionStableId")
+    Optional<Long> findParentQuestionIdByChildQuestionStableId(@Bind("childQuestionStableId") String childQuestionStableId);
```

Comment:
not used. used it initially and changed to other new method. left it as additional convenience method for any future use.